src/test/java/com/exasol/adapter/jdbc/ColumnMetadataReaderTest.java

```diff
@@ -42,27 +45,23 @@
 
     @BeforeEach
     void beforeEach() throws SQLException {
-        MockitoAnnotations.initMocks(this);
         when(this.connectionMock.getMetaData()).thenReturn(this.remoteMetadataMock);
     }
 
     @Test
     void testMapColumnsSingleColumn() throws SQLException {
-        mockDatatype(Types.BOOLEAN);
-        final ColumnMetadata column = mapSingleMockedColumn();
+        when(this.columnsMock.getInt(DATA_TYPE_COLUMN)).thenReturn(Types.BOOLEAN);
+        final ColumnMetadata column = mapSingleMockedColumn("BOOLEAN");
         assertAll(() -> assertThat(column.getName(), equalTo(COLUMN_A)),
                 () -> assertThat(column.getType(), equalTo(DataType.createBool())),
                 () -> assertThat(column.isNullable(), equalTo(true)),
                 () -> assertThat(column.isIdentity(), equalTo(false)));
     }
 
-    private void mockDatatype(final int typeId) throws SQLException {
-        when(this.columnsMock.getInt(BaseColumnMetadataReader.DATA_TYPE_COLUMN)).thenReturn(typeId);
-    }
-
-    private ColumnMetadata mapSingleMockedColumn() throws SQLException {
+    private ColumnMetadata mapSingleMockedColumn(final String typeName) throws SQLException {
         when(this.columnsMock.next()).thenReturn(true, false);
-        when(this.columnsMock.getString(BaseColumnMetadataReader.NAME_COLUMN)).thenReturn(COLUMN_A);
+        when(this.columnsMock.getString(NAME_COLUMN)).thenReturn(COLUMN_A);
```

Comment:
Speaking from experience I recommend putting the column type into the column name too (e.g. `BOOLEAN_COLUMN`. This makes debugging error messages easier.